### ISSUE TITLE
Incorrect documentation for search method/endpoint

### ISSUE THREAD

**Issue by bernEsp** [erikgrueter1/yelp-fusion#8]
In [Search API wiki](https://github.com/erikgrueter1/yelp-fusion#search-api) the examples are inaccurate.

```ruby 
params = { term: 'food',
           limit: 3,
           category_filter: 'discgolf'
         }

locale = { lang: 'fr' }

client.search('San Francisco', params, locale)
```

This example caused an error

```ruby
# error
ArgumentError: wrong number of arguments (3 for 1..2)
```

There are issues with `locale = { lang: 'fr' }`. The current code does not support or thread different this option to support and the third argument and the new search endpoint actually calls this option as `locale` instead of `lang`.

**Comment by erikgrueter1** [comment]
Hey @bernardogalindo Is the API working for you as expected now?

**Comment by bernEsp** [comment]
@erikgrueter1 Yes It is at least for this endpoint, we could close this issue due to we already merged #9
